app/admin.py

```diff
@@ -108,6 +109,8 @@ def post(self):
                         'deactivation_message_html',
                         'keywords',
                         'launch_status',
+                        'admin_user_email',
+                        'thres_unreviewed_notes',
```

Comment:
I believe you need to put this to json_config_names instead to make its type integer, not string.